plugins/dai-release/src/components/MetaModalPopupComponent/MetaModalPopupComponent.tsx

```diff
@@ -0,0 +1,214 @@
+import Button from '@mui/material/Button';
+import CircularProgress from '@mui/material/CircularProgress';
+import CloseIcon from '@material-ui/icons/Close';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import React from 'react';
+
+import { styled } from '@mui/material/zero-styled';
+import { useGetTemplateMetaInfo } from '../../hooks/useTemplatesMetaInfo';
+
+type modalPopupProps = {
+  openModal: boolean;
+  onClose: () => void;
+  instance: string;
+  modalPopupInputId: string;
+  modalTitle: string;
+  modalPopupData: any;
+  setModalPopupData: (data: any) => void;
+};
+
+export function MetaModalPopupComponent({
```

Comment:
Rename has popupComponent

Reply:
As per the discussion modified to "ModalComponent"